src/SFA.DAS.AssessorService.Web/Controllers/HomeController.cs

```diff
@@ -33,6 +33,9 @@ public HomeController(IDistributedCache cache, ISessionService sessionService, I
         [Route("/")]
         public IActionResult Index()
         {
+            // store the UseGovSignIn property value in the Session so that it could be used in the layout.
+            _sessionService.Set(nameof(WebConfiguration.UseGovSignIn), _webConfiguration.UseGovSignIn);
```

Comment:
Storing the UseGovSignIn value in the session and read it from Layout would be best choice rather than reading the config everytime. let me know your thoughts.